gtk4/src/subclass/im_context.rs

```diff
@@ -67,6 +67,11 @@ pub trait IMContextImpl: IMContextImplExt + ObjectImpl {
     fn activate_osk(&self) {
         self.parent_activate_osk()
     }
+    #[cfg(feature = "v4_14")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v4_14")))]
+    fn activate_osk_with_event(&self, event: Option<gdk::Event>) -> bool {
```

Comment:
```suggestion
    fn activate_osk_with_event(&self, event: Option<&gdk::Event>) -> bool {
```

etc. It's all transfer none, so no need to pass an owned value around